spec/services/slack_user_finder_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+describe SlackUserFinder do
+  describe "#user_exists?" do
+    it "returns true if a user is found" do
+      slack_username_from_fixture = "testusername"
+
+      user_exists = SlackUserFinder.new(
+        slack_username_from_fixture,
+        Slack::Web::Client.new
+      ).user_exists?
+
+      expect(user_exists).to be true
+    end
+
+    it "returns false if a user was not found" do
+      fake_slack_user_name = "testusernamethatdoesnotexist"
+
+      user_exists = SlackUserFinder.new(
+        fake_slack_user_name,
+        Slack::Web::Client.new
+      ).user_exists?
+
+      expect(user_exists).to be false
```

Comment:
another way to write this, if we keep the method name as a boolean, would be:


```ruby
user = SlackUserFinder.new(fake_slack_user_name, Slack::Web::Client.new)

expect(user).to be_user_exists
```
one thing I like about doing that is it reveals the quality of boolean method names - now that I wrote it this way, it seems the name of this method is repetitive / off. If we keep it a boolean, perhaps something like `is_in_organization?` would be better...

Source :https://www.relishapp.com/rspec/rspec-expectations/v/2-2/docs/matchers/predicate-matchers#matcher-arguments-are-passed-on-to-the-predicate-method